punchcut/src/source/glyf/math.rs

```diff
@@ -0,0 +1,83 @@
+//! Various fixed point math functions used in TrueType scaling and hinting.
```

Comment:
Needs comments and tests. What is it for, why is it here, why is our transform missing dx, dy, etc.

Also ... is this really only useful to punchcut? - it feels slightly misplaced, it's a fundamental buried in a task-specific crate. Should we have a font-math?

Reply:
Most of these are very specific to TrueType scaling and hinting. The transform function, for example, only deals with a 2x2 matrix (because that is the only thing present in a composite glyph) and has an additional scale factor to handle the mixture of 16.16 and 26.6 values.

I'll add comments and tests.

Reply:
The internal code now uses a new `F26Dot6` type in font-types and this module has been removed entirely. The small bits of single use code have been moved inline and commented.